<launch>
  <arg name="competition_id" default="0" /> <!-- 0 for semis, 1 for finals -->
  <arg name="loader_timer" default="5" />

  <node pkg="riptide_autonomy" type="be_autonomous" name="be_autonomous" output="screen" >
    <param name="competition_id" type="int" value="$(arg competition_id)" />
    <rosparam file="$(find riptide_autonomy)/cfg/task_execution.yaml" command="load" />
    <param name="loader_timer" type="double" value="$(arg loader_timer)" />
  </node>
</launch>
